src/main/java/org/checkstyle/autofix/CheckstyleCheck.java

```diff
@@ -38,9 +38,7 @@ public String getId() {
 
     public static Optional<CheckstyleCheck> fromSource(String source) {
         return Arrays.stream(values())
-                .filter(check -> check.getId().equals(source))
+                .filter(check -> check.getId().equals(source) || check.getId().contains(source))
```

Comment:
`check.getId().contains(source)` is enough, it already covers `check.getId().equals(source)`